libs/designer-ui/src/lib/templates/images/emptySearch.tsx

```diff
@@ -0,0 +1,118 @@
+export const EmptySearch: React.FC = () => {
```

Comment:
What is this image?

Reply:
![image](https://github.com/Azure/LogicAppsUX/assets/144840522/8664cf23-c3a6-4fd0-843b-ee9becc372a3)

It is for the empty search result, this svg image I got from Bonica